ini/rthun2.ini

```diff
@@ -1,41 +1,44 @@
-[data]
-name     = Rolling Thunder 2
-driver   = System2x
-type     = System2
-filename = snd0.bin
-filename = snd1.bin
-gain     = 1.0
-muterear = 0
-
-[wave.0]
-filename = voi1.bin
-length   = 0x80000
-position = 0x100000
-
-[wave.1]
-filename = voi2.bin
-length   = 0x80000
-position = 0x300000
-
-[playlist]
-04d = Attract
-032 = Title
-; 1003 = Credit
-028 = Round Start (Ver.1)
-016 = Theme from Rolling Thunder 2 (Round 1)
-01c = A Residence of Terrible (Round 2)
-022 = Across the Tube (Round 3)
-01f = Where is the Target? (Round 4)
-011 = Round Clear (Short Ver)
-013 = Game Over
-044 = Name Entry
-041 = Gimdo Goes To Egypt (Unused)
-050 = Mission 2 Start (Unused)
-02b = Round Start (Ver.2)
-119 = View of Dynamism (Round 5)
-12e = An Ancient Way (Round 6)
-125 = Terror for Sale (Round 7)
-13b = Running Decayed (Round 8)
-13e = The Final Command (Round 8)
-138 = Theme from Gimdo (Round 8)
-14a = Ending
+[data]
+name     = Rolling Thunder 2
+driver   = System2x
+type     = System2
+filename = snd0.bin
+filename = snd1.bin
+gain     = 1.0
+muterear = 0
+
+[wave.0]
+filename = voi1.bin
+length   = 0x80000
+position = 0x100000
+
+[wave.1]
+filename = voi2.bin
+length   = 0x80000
+position = 0x300000
+
+[playlist]
+04d = ATTRACT

```

Comment:
Not sure what's the point of capitalizing everything here. I prefer to keep mixed case if the OST track names are all uppercase

Reply:
Yeah, I was just going off of the soundtrack releases. I can change it back.